torch/csrc/jit/passes/xnnpack_rewrite.cpp

```diff
@@ -294,6 +296,15 @@ script::Module optimizeForMobile(
     FoldPrePackingOps(cloned_module);
   }
 
+  if (!optimization_blocklist.count(
+          MobileOptimizerType::HOIST_CONV_PACKED_PARAMS)) {
+    // freeze again in case it was not done in previous optional passes
+    cloned_module = freeze_module(cloned_module, preserved_methods);
```

Comment:
I'm seeing optimize_for_mobile test fail on this line (https://gist.github.com/vkuzo/0e0562fd46a10b9ed8f2f6e8eefca349), just checking that this is expected and we can update the test to match?  seems like that test case currently assumes the BN::forward call survives

Reply:
No I dont think so.  BNTestModule has BN and Conv2d. The test where it fails, it has disabled prepack folding. If that is disabled there is no inlining, via freeze_module. So we should still have Conv2d::forward. I think that is what it is expecting. But since you have freeze_module after, and you probably will add freeze_module before, inlining will happen and there wont be any forward calls. So yeah we will have to fix that test somehow. Maybe best to remove that portion of the test.